templates/datadog_footer.conf.erb

```diff
@@ -347,6 +347,11 @@ process_agent_enabled: <%= @process_enabled %>
 # Enable the trace agent.
 apm_enabled: <%= @apm_enabled %>
 
+# Enable and the trace search & analytics by adding a list of entered APM events.
+  analyzed_spans:
+    <%= @apm_analyzed_spans.each { |service, resource, percentage|
+            service.nil? || resource.nil? || percentage.nil? ? (puts "Error data missing for: #{service}, #{resource}, #{percentage}") : (raise "#{service}|#{resource}: #{percentage}")
```

Comment:
Though this looks good, it would probably be unnecessary if we define the strict data type for `apm_analyzed_spans` as done above.

Reply:
Have minimized the insertions of the data to the template file.